files/openhim/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3'
+ 
+services:
+  mongo:
+    container_name: mongo
+    image: mongo:3.4
+    ports:
+      - "27017:27017"
+    volumes:
+      - openhim-data:/data/db
+    restart: always
+    healthcheck:
+      test: echo 'db.stats().ok' | mongo localhost:27017/test --quiet
+      interval: 1m
+      timeout: 3s
+      retries: 3
+ 
+  openhim-core:
+    container_name: openhim-core
+    image: jembi/openhim-core
+    environment:
+      mongo_url: mongodb://mongo/openhim
+      mongo_atnaUrl: "mongodb://mongo/openhim"
+    ports:
```

Comment:
Which ports should I expose on the host (from 80/443?) 
Or do you want all of them, using a different DNS name each?

Reply:
Oh, ignore it. Now I see there's one more container below :D